tests/utils/test_serialization.py

```diff
@@ -89,13 +84,7 @@ class Model(BaseModel):
     conset: pydantic.conset(int) = ConstrainedSet([1])  # type: ignore[valid-type]
     confrozenset: pydantic.confrozenset(int) = ConstrainedFrozenSet([1])  # type: ignore[valid-type]
     conint: pydantic.conint() = ConstrainedInt(1)  # type: ignore[valid-type]
-    conlist: pydantic.conlist(str) = ConstrainedList([1])  # type: ignore[valid-type]
-
-    strict_str: StrictStr = StrictStr("hello")
-    strict_int: StrictInt = StrictInt(1)
-    strict_float: StrictFloat = StrictFloat(1.0)
-    strict_bytes: StrictBytes = StrictBytes(b"hello")
-    strict_bool: StrictBool = StrictBool(True)
```

Comment:
We don't need to test for those explicitly, since they are covered by their base types